src/App.js

```diff
@@ -46,6 +51,11 @@ export default function App() {
         <Route path="/success">
           <SuccessPage />
         </Route>
+        <Route path="/schedule"><Schedule /></Route>
```

Comment:
If placed here, the user can access these routes even if they aren't logged in. 

To fix this, You could either put these routes in Home.jsx, or put them in the isUserLoggedIn() fucntion

Reply:
I understand they should be moved, but I'm a little lost on where in isUserLoggedIn() they should be placed. After a bit of research, it looks like [ProtectedRoutes](https://ui.dev/react-router-v4-protected-routes-authentication/) are the standard for this kind of issue. Would you like me to set that up and push it Saturday or Sunday, or would you rather have them placed in isUserLoggedIn()?

Reply:
Ive never heard of ProtectedRoutes but they look very easy to use and applicable to this situation. If you want to implement it like that then definitely go for it. After pushing those changes, this should be good to be merged!

Reply:
I just added the protected routes so everything should be good to go now